### DISCUSSION TITLE
Obsidian.md Integration/Exporting

### DISCUSSION THREAD

**Discussion by Gruffel** [dailydotdev/daily#847] [Feature Requests]
Building a connection with Obsidian.md.

This tooling could be simply for quick exporting articles meta data, quick references, and personal notes.
Maybe share Obsidian notes in squad or in the comments of the daily.dev summaries.

Maybe build a plugin to get your daily.dev account to be built into an Obsidian.md Vault or be a Vault? 
(Synced with the extension, and App versions)

**Comment by idoshamun** [comment]
Hi @Gruffel,

This can be a great idea for future integrations once we focus on providing external API.
Thanks for the suggestion!
